audio.js

```diff
@@ -1,12 +1,225 @@
+//Display audio element id
 const display = document.getElementById("display");
 
-$(".playback").click(function (e) {
-    e.preventDefault();
+//Power ON and OFF
+let toggle = document.getElementById("power-switch-input");
+display.innerHTML = 'Power Off';
 
-    var clip = $(this).next('audio').get(0);
+$('.playback').attr('disabled', 'disabled'); //audio off
 
-    display.innerHTML = clip.id;
+// console.log(typeof toggle.value);
 
-    clip.currentTime = 0;
-    clip.play();
+toggle.addEventListener('input', () => {
+    if ((toggle.value == '0' && bank.value == '0') || (toggle.value == '0' && bank.value == '1')) {
+        toggle.value = '1';
+
+        display.innerHTML = "Power On";
+
+        async function keySounds(key, id) {
```

Comment:
This function can be moved outside of this function, to the top level of the file and doesn't need to be async. See the current audio.js file on our master branch.

Reply:
Wait sorry I forgot this handles keypresses. This can be rewritten so that, when the user presses a key, it `click`s the corresponding element. That way the `onclick` event only needs to be written once. We can handle this later.

Reply:
after testing it again, the words "async" and "await" can be removed now because my "if(clip == null)" statement resolved the dom errors.

**"This can be rewritten so that, when the user presses a key, it clicks the corresponding element"**, thats what the keySounds function does, so not following your logic here.

Reply:
>This can be rewritten so that, when the user presses a key, it clicks the corresponding element

>thats what the keySounds function does, so not following your logic here.

I was referring to the function below, and line 36 in particular. Instead of writing two functions which handle the playing of audio samples (lines 36 and 137), we can write one function to handle when a button is clicked (137) and in this function we can `button.click()` to invoke that function.